alignment/calibrationsquare.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from alignment.calibrationsquareobservation import CalibrationSquareObservation
+from scipy.spatial.transform import Rotation as R
```

Comment:
For someone reading the code it's clearer to use the full word `Rotation` rather than the abbreviation `R`, which could stand for lots of things (that begin with "r")